reason-react-native/src/apis/CameraRoll.md

```diff
@@ -41,6 +40,8 @@ The function will return a `photoIdentifiersPage` object wrapped in a Promise, t
 ## Types
 
 - `getPhotosParams` can be constructed with its eponymous constructor
```

Comment:
Had to look up "eponymous" in the dictionary. 😂 Maybe use simpler wording for non-native speakers?

"with its constructor of the same name"?

Reply:
It was fun to use though :)